src/helpers/get-int-config.ts

```diff
@@ -0,0 +1,21 @@
+import { InvalidConfigException } from '@ukef/config/invalid-config.exception';
+
+// This helper function is used to get integer from configuration.
+export const getIntConfig = (environmentVariable: string, defaultValue?: number): number | undefined => {
+  if (typeof environmentVariable === 'undefined') {
+    if (typeof defaultValue === 'undefined') {
+      throw new InvalidConfigException(`Environment variable is missing and doesn't have default value.`);
+    }
+    return defaultValue;
```

Comment:
Do we need to check that `defaultValue` is an int?

Reply:
I wanted to avoid complicating this too much. Also typescript provides some validation ensuring that `defaultValue` is number.